src/rust/ensogl/src/display/symbol/gpu/registry.rs

```diff
@@ -38,6 +38,7 @@ pub struct SymbolRegistry {
     symbol_dirty    : SymbolDirty,
     logger          : Logger,
     view_projection : Uniform<Matrix4<f32>>,
+    z_zoom_1        : Uniform<f32>,
```

Comment:
It might be useful to re-iterate part of the doc from the `Camera2D` about what the `z_zoom_1` value does and why it is called that: ("The `z_zoom_1` value describes the z-axis distance at which the `zoom` value is `1.0`"). My first impression was that it should just be called `z_zoom`, but with the description, the name makes more sense.

Reply:
Added comment about it. I think after the release I would need to spend few days refactoring this part as its getting messy slowly